docs/source/blueprints/template.rst

```diff
@@ -0,0 +1,86 @@
+BP???
+#####
```

Comment:
Would be nice to have a reference to BP000 on this template (as a comment, maybe)

Reply:
Yeah, good point.